crates/bevy_render/src/camera/camera.rs

```diff
@@ -212,17 +212,43 @@ impl Camera {
         Some((ndc_space_coords.truncate() + Vec2::ONE) / 2.0 * target_size)
     }
 
+    /// Returns a ray originating from the camera, that passes through everything beyond `viewport_position`.
+    ///
+    /// The resulting ray starts on the near plane of the camera.
+    ///
+    /// If the camera's projection is orthographic the direction of the ray is always equal to `camera_transform.forward()`.
+    ///
+    /// To get the world space coordinates with Normalized Device Coordinates,
+    /// you should use [`ndc_to_world`](Self::ndc_to_world).
+    pub fn viewport_to_world(
```

Comment:
This method doesn't account for the `scale` field of an orthographic projection. Is that going to be a problem?

Reply:
That should be accounted for in `self.computed.projection_matrix` which is used in `ndc_to_world` I think.